clipboard/src/web.ts

```diff
@@ -0,0 +1,100 @@
+import { WebPlugin } from '@capacitor/core';
+import {
+  ClipboardPlugin,
+  ClipboardWrite,
+  ClipboardReadResult,
+} from './definitions';
+
+declare var navigator: any;
+declare var ClipboardItem: any;
+
+export class ClipboardWeb extends WebPlugin implements ClipboardPlugin {
+  constructor() {
+    super({
+      name: 'Clipboard',
+      platforms: ['web'],
+    });
+  }
+
+  async write(options: ClipboardWrite): Promise<void> {
+    if (!navigator.clipboard) {
+      return Promise.reject('Clipboard API not available in this browser');
```

Comment:
Within `async` functions, we prefer to use `throw new Error(msg)` syntax, which is equivalent to `return Promise.reject(new Error(msg))`. 

Additionally, when the current browser is unsupported, we want to throw the new `UnsupportedBrowserException` error. See https://github.com/ionic-team/capacitor-plugins/pull/1/files#diff-cfdb994cb3dbedf92d7eab756a4de2ccR17-R19.